ush/ioda/bufr2ioda/bufr2ioda_ncep_satwind_amv_goes.py

```diff
@@ -0,0 +1,530 @@
+# (C) Copyright 2023 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+import sys
+
+sys.path.append('/work/noaa/da/eliu/JEDI-iodaconv/ioda-bundle/build/lib/')
```

Comment:
remove hard coded paths for orion, my shell script should handle this properly (I hope)